pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/OfflineClusterIntegrationTest.java

```diff
@@ -236,7 +236,7 @@ public void testRefreshTableConfigAndQueryTimeout()
       throws Exception {
     // Set timeout as 5ms so that query will timeout
     TableConfig tableConfig = getOfflineTableConfig();
-    tableConfig.setQueryConfig(new QueryConfig(5L));
+    tableConfig.setQueryConfig(new QueryConfig(5L, 2));
```

Comment:
This seems like a change from current behavior. Any reason why we want to constrain the number of cores to be used to 2 for integration tests?

Reply:
@mayankshriv  I am not that familiar with integration tests code. There is not specific reason to use value of 2 here. Pls suggest a reasonable value that can be used here. thanks